app/controllers/static_pages_controller.rb

```diff
@@ -1,7 +1,7 @@
 class StaticPagesController < ApplicationController
-  def home
-  end
+  def home;end
```

Comment:
```suggestion
  def home; end
```
sau dấu ; cách 1 space ra em